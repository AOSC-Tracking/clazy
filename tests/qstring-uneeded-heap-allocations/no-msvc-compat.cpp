#include <QtCore/QString>
#include <QtCore/QStringList>

static QStringList list = { "foo" };

void test()
{
    QString s = "foo";
}
