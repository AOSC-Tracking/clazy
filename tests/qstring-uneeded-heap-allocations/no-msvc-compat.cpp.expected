qstring-uneeded-heap-allocations/no-msvc-compat.cpp:4:29: warning: QString(const char*) being called [-Wclazy-qstring-uneeded-heap-allocations]
qstring-uneeded-heap-allocations/no-msvc-compat.cpp:8:17: warning: QString(const char*) being called [-Wclazy-qstring-uneeded-heap-allocations]
